src/schema/v2/homeView/sections.ts

```diff
@@ -50,6 +52,7 @@ export type HomeViewSection = {
 export const SimilarToRecentlyViewedArtworks: HomeViewSection = {
   id: "home-view-section-similar-to-recently-viewed-artworks",
   type: HomeViewSectionTypeNames.HomeViewSectionArtworks,
+  contextModule: ContextModule.similarToWorksYouViewedRail,
```

Comment:
**thought:** At first, introducing the dependency on Cohesion felt unnatural to me, but tbh if MP is the authority now on what appears in the UI, this makes sense to me and serves our goal of enforcing valid analytics payloads. Works for me 👍🏽